### DISCUSSION TITLE
Cor drawBlackBox

### DISCUSSION THREAD

**Discussion by jvolima** [404jv/dev-landia#90] [Q&A] [answered]
@404jv @FixRuan A cor do drawBlackBox está muito parecida com a do fundo, de forma que fica complicado de enxergar, o que vocês preferem, alterar a cor do fundo onde fica os draws, ou mudar a cor do drawBlackBox?

**Comment by 404jv** [comment, marked as answer]
Mudar a cor do drawBlackBox. Até porque conforme os exercícios for evoluindo, a cor dos comandos no editor vão ser apenas algumas. A fim de ser mais próximo possível de um editor de código real.
